QuizApp/User/UserUseCase.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+protocol UserUseCaseProtocol {
+
+    func getUser() async throws -> UserResponseModel
+
+}
+
+class UserUseCase: UserUseCaseProtocol {
+
+    private var datasource: UserDatasource!
+    private var tokenStorage: SecureStorage!
+
+    init(tokenStorage: SecureStorage) {
+        self.tokenStorage = tokenStorage
+        self.datasource = UserDatasource()
+    }
+
+    func getUser() async throws -> UserResponseModel {
+        return try await datasource.getUser(accessToken: tokenStorage.accessToken ?? "")
```

Comment:
You don't need `return`.
And you can throw error here if there's no `accessToken` in the `tokenStorage`.